rocketpy/utilities.py

```diff
@@ -654,3 +655,25 @@ def liftoff_speed(mass):
     if plot:
         retfunc.plot(min_mass, max_mass, points)
     return retfunc
+
+
+def get_instance_attributes(instance):
```

Comment:
Did you consider using the built-in `vars()` from Python? See [link](https://docs.python.org/3/library/functions.html#vars).
This would allow you to get the __dict__ of the instance and the code would become much easier to understand.

Reply:
Unfortunately this does not work with `@funcify_method` and `@cached_properties`